package.json

```diff
@@ -275,13 +276,14 @@
     "bignumber.js": "^4.1.0",
     "bn.js": "^4.11.7",
     "bowser": "^2.11.0",
+    "btoa": "^1.2.1",
     "classnames": "^2.2.6",
     "copy-to-clipboard": "^3.0.8",
     "currency-formatter": "^1.4.2",
     "debounce-stream": "^2.0.0",
     "deep-freeze-strict": "1.1.1",
     "end-of-stream": "^1.4.4",
-    "eth-block-tracker": "^6.0.0",
+    "eth-block-tracker": "7.0.0",
```

Comment:
Can we use `^` here?

Reply:
done here: https://github.com/MetaMask/metamask-extension/pull/17652/commits/4e3911a00695766a0edef23a4958edc66779bb24